spec/OpenLineage.json

```diff
@@ -108,7 +113,14 @@
           "description": "The job facets.",
           "type": "object",
           "additionalProperties": {
-            "$ref": "#/$defs/JobFacet"
+            "anyOf": [
+              { "$ref": "#/$defs/JobFacet" },
+              { "$ref": "facets/DocumentationJobFacet.json" },
```

Comment:
I think this should be: 
```
{ "$ref": "facets/DocumentationJobFacet.json#/$defs/DocumentationJobFacet" },
```
same for other facets

Reply:
So that it points to this object: https://github.com/OpenLineage/OpenLineage/blob/main/spec/facets/DocumentationDatasetFacet.json#L5